test/helpers.bash

```diff
@@ -195,3 +163,29 @@ calculate_pods_per_node() {
     echo "75"
   fi
 }
+
+verify_object_count(){
+  local resource=$1
+  local count=$2
+  local namespace=$3
+  local selector=$4
+  local fieldSelector=$5
+  local CMD="oc get $1 --no-headers"
```

Comment:
I know this repo references `ocp`. But, the code you deleted used `kubectl`. Should we keep using the latter?